ssvsigner/cmd/ssv-signer/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"log"
+
+	"github.com/alecthomas/kong"
+	"github.com/ssvlabs/ssv/operator/keys"
+	"github.com/ssvlabs/ssv/operator/keystore"
+	"github.com/valyala/fasthttp"
+	"go.uber.org/zap"
+
+	"github.com/ssvlabs/ssv/ssvsigner/server"
+	"github.com/ssvlabs/ssv/ssvsigner/web3signer"
+)
+
+type CLI struct {
+	ListenAddr              string `env:"LISTEN_ADDR" default:":8080"` // TODO: finalize port
+	Web3SignerEndpoint      string `env:"WEB3SIGNER_ENDPOINT"`
+	PrivateKey              string `env:"PRIVATE_KEY"`
+	PrivateKeyFile          string `env:"PRIVATE_KEY_FILE"`
+	PasswordFile            string `env:"PASSWORD_FILE"`
+	ShareKeystorePassphrase string `env:"SHARE_KEYSTORE_PASSPHRASE" default:"password"` // TODO: finalize default password
+}
+
+func main() {
+	cli := CLI{}
+	_ = kong.Parse(&cli)
+
+	logger, err := zap.NewDevelopment()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer func() {
+		if err := logger.Sync(); err != nil {
+			log.Println("failed to sync logger", zap.Error(err))
+		}
+	}()
```

Comment:
The `logger.Sync()` call inside `defer` might return an error that isn't logged properly because `zap.Error(err)` works differently with `log.Println`.  

Suggested fixes:  
- Use structured logging with Zap:  
  ```go
  logger.Warn("failed to sync logger", zap.Error(err))
  ```
- Use correct formatting with `log.Println`:
  ```go
  log.Println("failed to sync logger:", err.Error())
  ```

Reply:
fixed